### PR TITLE
Roles and edit buttons are separated

### PR DESCRIPTION
**Pull request by stephdl**
- Display two buttons for role and edit
- Roles is always visible for root
- Edit is displayed only if the account provider is installed locally

NethServer/dev#6023

### CODE REVIEW

**Comment by stephdl** [comment]
On local server

Screenshot_2020-02-03 Users Groups - ns7loc9 nethservertest org(2)
Screenshot_2020-02-03 Users Groups - ns7loc9 nethservertest org(1)
Screenshot_2020-02-03 Users Groups - ns7loc9 nethservertest org

**Comment by stephdl** [comment]
on the remote cockpit

Screenshot_2020-02-03 Users Groups - ns7loc13 nethservertest org(1)
Screenshot_2020-02-03 Users Groups - ns7loc13 nethservertest org

**Comment by stephdl** [comment]
Screenshot_2020-02-04 Users Groups - ns7loc9 nethservertest org
Screenshot_2020-02-04 Users Groups - ns7loc13 nethservertest org(1)
Screenshot_2020-02-04 Users Groups - ns7loc13 nethservertest org

File: ui/src/components/system/UsersGroups.vue [hunk 1]
```diff
@@ -579,19 +589,25 @@
...
           <div class="modal-header">
-            <h4 class="modal-title">
+            <h4 v-if="newGroup.EditGroup" class="modal-title">
               {{newGroup.isEdit ? $t('edit') : $t('users_groups.create_group')}}
               <span
                 v-if="newGroup.isEdit"
               >{{newGroup.name}}</span>
             </h4>
+            <h4 v-else class="modal-title">
+              {{$t('users_groups.edit_delegation')}}
```
Comment (DavidePrincipi) [inline ui/src/components/system/UsersGroups.vue]:
```suggestion
              {{$t('users_groups.edit_delegation', {name: newGroup.name})}}
```

For good i18n the strings must use placeholders as the position of "name" (in this case) depends on the language. We cannot assume the correct position of *name* is after the *edit_delegation* label in every possible language. 

A similar rule applies to punctuation: periods and question marks must be in translated strings. 
/cc @edospadoni @gsanchietti 

See also
- https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting
- https://www.gnu.org/software/gettext/manual/html_node/Preparing-Strings.html#Preparing-Strings

Comment (stephdl) [reply ui/src/components/system/UsersGroups.vue]: Put objects, vars in translation is a really bad thing, see what it occurred in French

Comment (DavidePrincipi) [reply ui/src/components/system/UsersGroups.vue]: Translators are as programmers: they make errors. Sometimes they don't even know what they are typing :D

Comment (stephdl) [reply ui/src/components/system/UsersGroups.vue]: ok

**Pull request merged**